.neuro/live.yml

```diff
@@ -48,10 +48,9 @@ images:
 
 jobs:
   create_pipeline:
-    preset: cpu-small
-    pass_config: True
-    detach: True
+    name: $[[ flow.title ]]-create-pipeline
     image: $[[ images.train.ref ]]
+    pass_config: True
     volumes:
       - secret:gh-rsa:/root/.ssh/id-rsa
     params:
```

Comment:
How about we add a parameter with pipeline name?

Reply:
sounds good, will do that

Reply:
done in e7f9a7d